infrastructure/environments/setup-environment.ts

```diff
@@ -718,6 +718,25 @@ const derivedVariables = [
   }
 ] as const
 
+const metabaseAdminQuestions = [
+  {
+    valueType: 'SECRET',
+    name: 'OPENCRVS_METABASE_ADMIN_EMAIL',
+    type: 'text',
+    message: 'Email for metabase admin dashboard login',
```

Comment:
Verify these questions are asked properly when user runs the environment creation script. 

Let's also explain the variables a bit more so 

```
Email for Metabase super admin. Used as a username when logging in to the dashboard.
```

Reply:
@rikukissa done https://github.com/opencrvs/opencrvs-countryconfig/pull/128/commits/ed89e54d9775242d20acfa87cc9e62ef8d039295